src/WebEncoders/src/Testing/HtmlTestEncoder.cs

```diff
@@ -12,11 +12,13 @@ namespace Microsoft.Extensions.WebEncoders.Testing
     /// </summary>
     public sealed class HtmlTestEncoder : HtmlEncoder
     {
+        /// <inheritdoc />
```

Comment:
Could we enhance the summary for this type to say something about not using it in production apps?

Reply:
Same for the other test encoders

Reply:
The type summary already indicates that it's used for testing only. Didi you want to add it to all the method docs as well?

Reply:
It feels a bit spare. How about 

> `<see cref="HtmlEncoder" /> used for unit testing. This encoder does not perform any encoding and should not be used in application code.

Reply:
Updated for all three encoders.